.github/workflows/release.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Install dependencies
         shell: bash
         run: |
-          yarn install --frozen-lockfile
+          yarn install
```

Comment:
we don't want to release with a frozen lock file? we want the random internet resolution to take effect on every release ?

Reply:
`--frozen-lockfile` has been replace with `--immutable` which is automatically enabled when it detects a CI environment,